Policies/Kubernetes/host-network-ports/rego/src_test.rego

```diff
@@ -0,0 +1,161 @@
+package k8spsphostnetworkingports
+
+test_input_no_hostnetwork_no_port_is_allowed {
+    input := { "review": input_review, "parameters": input_parameters_ports}
```

Comment:
Can you add a test for input_review with input_parameters_ports_no_hostnetwork ?